mountinfo/mountinfo_windows.go

```
@@ -0,0 +1,6 @@
+package mountinfo
+
+func parseMountTable() ([]*Info, error) {
+	// Do NOT return an error!
+	return nil, nil
```

This should probably return an error.